### PR TITLE
Feature/jude/user sign in

### PR DESCRIPTION
**Pull request by judeokafor**
fixed lint errors from previous code.
User sign in route coupled
User Authorization using token from sign up method

### CODE REVIEW

**Comment by enyeInc** [comment]
@judeokafor PR is too large. Break it into smaller parts

File: functions/src/utils/firebase.js [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+const firebaseConfig = {
+    apiKey: config.firebaseConfig.apiKey,
+    appId: config.firebaseConfig.appId,
+    authDomain: config.firebaseConfig.authDomain,
+    databaseURL: config.firebaseConfig.databaseURL,
+    measurementId: config.firebaseConfig.measurementId,
+    messagingSenderId: config.firebaseConfig.messagingSenderId,
+    projectId: config.firebaseConfig.projectId,
+    storageBucket: config.firebaseConfig.storageBucket,
+};
```
Comment (Simpcyclassy) [inline functions/src/utils/firebase.js]: Could you destruct all the config.firebaseConfig, I think it is quicker to read that way

File: functions/src/utils/firebaseAdmin.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    auth_uri: config.adminConfig.auth_uri,
+    client_email: config.adminConfig.client_email,
+    client_id: config.adminConfig.client_id,
+    client_x509_cert_url: config.adminConfig.client_x509_cert_url,
+    private_key: config.adminConfig.private_key.replace(/\\n/g, '\n'),
+    private_key_id: config.adminConfig.private_key_id,
+    project_id: config.adminConfig.project_id,
+    token_uri: config.adminConfig.token_uri,
+    type: config.adminConfig.type,
+};
```
Comment (Simpcyclassy) [inline functions/src/utils/firebaseAdmin.js]: Same thing here, so you could just have
`auth_provider_x509_cert_url,`
`auth_uri,`

**Review by codemutatio — APPROVED** [review]
Looks good to me but subsequently please try and make sure your PR lines of code is not more than 250 lines.

**Pull request merged**